validator_client/src/cli.rs

```diff
@@ -466,6 +466,17 @@ pub struct ValidatorClient {
     )]
     pub beacon_nodes_sync_tolerances: Vec<u64>,
 
+    #[clap(
+        long,
+        help = "Enable the beacon head monitor so fallback head updates trigger duties when a lagging        primary is detected. \
```

Comment:
This line has too many spaces in it, we could probably flow it onto the next line,

```suggestion
        help = "Enable the beacon head monitor so fallback head updates trigger duties when a lagging primary is detected. \
```